riemann_client/client.py

```diff
@@ -152,6 +153,9 @@ def create_dict(event):
                 value = list(value)
             elif descriptor.name == 'attributes':
                 value = dict(((a.key, a.value) for a in value))
+            elif descriptor.type == \
+                    google.protobuf.descriptor.FieldDescriptor.TYPE_FLOAT:
+                value = round(value, 1)
```

Comment:
I've pushed a PR that should fix this (and makes some needed changes anyway) here: https://github.com/borntyping/python-riemann-client/pull/33. I'm merge it after this PR if you don't have any further changes, or close it if not needed.